### PR TITLE
Training script with one loss function

### PR DESCRIPTION
**Pull request by merryHunter**
Hi, I composed a script for training a network with one loss and slightly faster data gen.

### CODE REVIEW

**Comment by merryHunter** [comment]
PS: I have not tested the performance of the network  in CARLA.

**Comment by mvpcom** [comment]
I believe this can help others to achieve their goals faster. All changes need to be tested and integrated into the main code as soon as possible.
